tests/unit/cocoon/test_utilities.py

```diff
@@ -1587,24 +1587,44 @@ def test_identify_cash_items_failed(
                     }
                 },
                 False,
-                ["GBP_EXP", "GBP_EXP", "USD_EXP", "USD_EXP"],
-            ),
-            (
-                "combined_currency_code_inference",
-                {
-                    "cash_identifiers": {
-                        "instrument_name": {
-                            "inst1": "GBP_EXP",
-                            "inst2": "GBP_EXP",
-                            "inst3": "USD_EXP",
-                            "inst4": "",
-                        },
-                    },
-                    "implicit": "internal_currency",
-                },
-                False,
-                ["GBP_EXP", "GBP_EXP", "USD_EXP", "USD_IMP"],
-            ),
+                ["GBP_EXP", "GBP_EXP", "USD_EXP", "USD_EXP", None],
+            )
+        ]
+    )
+    def test_identify_cash_items_remove_false(self, _, cash_flag, remove_cash_items, ground_truth):
```

Comment:
couple of things:
1) its not clear from the name what this test is meant to be doing, i think it should be renamed to something like `test_remove_cash_items_with_identify_cash_items`
2) i think `ground_truth` is your expected values, rename to something like `expected_values`
3) `remove_cash_items` doesn't look like it needs to be passed in and can be set in the call - its `False` for all

Reply:
3. ✔️

Reply:
2. ✔️

Reply:
1, ✔️ :
`test_identify_cash_items_with_remove_cash_items()`
`test_identify_cash_items_without_remove()`

Reply:
there are still some "ground truths" in there, can you rename to "expected values" or something equivalent